src/core/client/stream/tabs/Comments/Stream/UnansweredCommentsTab/UnansweredCommentsTabContainer.tsx

```diff
@@ -0,0 +1,329 @@
+graphql`
+  fragment UnansweredCommentsTabContainer_comment on Comment {
+    id
+    ...CommentContainer_comment
+    ...ReplyListContainer1_comment
+    ...IgnoredTombstoneOrHideContainer_comment
+  }
+`;
+
+export const UnansweredCommentsTabContainer: FunctionComponent<
+  Props
+> = props => {
+  const [{ commentsOrderBy }] = useLocal<UnansweredCommentsTabContainerLocal>(
+    graphql`
+      fragment UnansweredCommentsTabContainerLocal on Local {
+        commentsOrderBy
+      }
+    `
+  );
+  const subscribeToCommentCreated = useSubscription(CommentCreatedSubscription);
+  const subscribeToCommentReleased = useSubscription(
+    CommentReleasedSubscription
+  );
+  useEffect(() => {
+    if (!props.story.settings.live.enabled) {
+      return;
+    }
+
+    if (props.story.isClosed || props.settings.disableCommenting.enabled) {
+      return;
+    }
+    if (
+      commentsOrderBy === GQLCOMMENT_SORT.CREATED_AT_ASC &&
+      props.relay.hasMore()
+    ) {
+      // If sort by oldest we only need to know if there is more to load.
+      return;
+    }
+    if (
+      ![
+        GQLCOMMENT_SORT.CREATED_AT_ASC,
+        GQLCOMMENT_SORT.CREATED_AT_DESC,
+        // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
+      ].includes(commentsOrderBy as GQLCOMMENT_SORT)
+    ) {
+      // Only chronological sort supports top level live updates of incoming comments.
+      return;
+    }
+    const newCommentDisposable = subscribeToCommentCreated({
+      storyID: props.story.id,
+      orderBy: commentsOrderBy,
+    });
+    const releasedCommentDisposable = subscribeToCommentReleased({
+      storyID: props.story.id,
+      orderBy: commentsOrderBy,
+    });
+    return () => {
+      newCommentDisposable.dispose();
+      releasedCommentDisposable.dispose();
+    };
+  }, [
+    commentsOrderBy,
+    subscribeToCommentCreated,
+    subscribeToCommentReleased,
+    props.story.id,
+    props.relay.hasMore(),
+    props.story.settings.live.enabled,
+  ]);
+  const [loadMore, isLoadingMore] = useLoadMore(props.relay, 20);
+  const beginLoadMoreEvent = useViewerNetworkEvent(LoadMoreAllCommentsEvent);
+  const loadMoreAndEmit = useCallback(async () => {
+    const loadMoreEvent = beginLoadMoreEvent({ storyID: props.story.id });
+    try {
+      await loadMore();
+      loadMoreEvent.success();
+    } catch (error) {
+      loadMoreEvent.error({ message: error.message, code: error.code });
+      // eslint-disable-next-line no-console
+      console.error(error);
+    }
+  }, [loadMore, beginLoadMoreEvent, props.story.id]);
+  const viewMore = useMutation(UnansweredCommentsTabViewNewMutation);
+  const onViewMore = useCallback(() => viewMore({ storyID: props.story.id }), [
+    props.story.id,
+    viewMore,
+  ]);
+  const comments = props.story.comments.edges.map(edge => edge.node);
+  const viewNewCount =
+    (props.story.comments.viewNewEdges &&
+      props.story.comments.viewNewEdges.length) ||
+    0;
+  return (
```

Comment:
i think this all makes sense but it is a lot

Reply:
this is a useless comment lol ignore.

Reply:
Eh, I agree, will do what I can. Was original copy pasted from Featured stream, but that doesn't mean we can't clean it up! 😄